tests/testsuite/profiles.rs

```diff
@@ -269,14 +269,14 @@ fn profile_in_non_root_manifest_triggers_a_warning() {
         .file("bar/src/main.rs", "fn main() {}")
         .build();
 
-    p.cargo("build -v")
+    p.cargo("check -v")
```

Comment:
I suggest not modifying anything in `profiles.rs` since they often involve codegen.